sigh/lib/sigh/runner.rb

```diff
@@ -216,7 +216,12 @@ def certificate_to_use
     # Downloads and stores the provisioning profile
     def download_profile(profile)
       UI.important "Downloading provisioning profile..."
-      profile_name ||= "#{profile_type.pretty_type}_#{Sigh.config[:app_identifier]}.mobileprovision" # default name
+      profile_name ||= "#{profile_type.pretty_type}_#{Sigh.config[:app_identifier]}"
```

Comment:
ok good point, yep thats fine, i was just curious about the infact usless unless 👍